packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -581,7 +583,9 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
 /**
  * SharedObject with simplified, synchronous summarization and GC.
  * DDS implementations with async and incremental summarization should extend SharedObjectCore directly instead.
- * @alpha
+ * @privateRemarks
+ * used by fluidframework/map public SharedMap
+ * @public
  */
 export abstract class SharedObject<
```

Comment:
Its possible for a DDS to not leak this as public (and instead just IChannel), see tree2 as an example. If we are going to stabilize another DDS, we should follow that pattern.

Reply:
Also, if we only make the aqueduct/azure entry point public (which I think is the plan?), I think we might be able to leave the factories (and thus IChannel, SharedObject etc) as alpha APIs and not public. This seems like it would be a good idea.

Reply:
azure will be public, and so will map

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/6592